odoo/tools/translate.py

```diff
@@ -1089,6 +1129,7 @@ def _export_translatable_resources(self):
         - the python strings marked with _() or _lt()
         - the javascript strings marked with _t() or _lt() inside static/src/js/
         - the strings inside Qweb files inside static/src/xml/
+        - the spreadsheet data files
```

Comment:
the spreadsheet dashboards data files

Reply:
actually it can be any spreadsheet. It could also be spreadsheet templates